ChemistryLib/PhreeqcInterface.h

```diff
@@ -0,0 +1,84 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2019, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#pragma once
+
+#include <memory>
+
+#include "BaseLib/ConfigTree.h"
+#include "MathLib/LinAlg/GlobalMatrixVectorTypes.h"
+#include "Output.h"
+#include "PhreeqcInterfaceData/AqueousSolution.h"
+#include "PhreeqcInterfaceData/EquilibriumPhase.h"
+#include "PhreeqcInterfaceData/KineticReactant.h"
+#include "PhreeqcInterfaceData/ReactionRate.h"
+
+namespace ChemistryLib
+{
+enum class Status
+{
+    SettingAqueousSolutions,
+    UpdatingProcessSolutions
+};
+
+class phreeqcIO
```

Comment:
File vs. class name mismatch.

Reply:
We have two options. One is to change the file name to `PhreeqcIO.h`| `PhreeqcIO.cpp`and the folder name to `PhreeqcIOData`. The other is to recast the class name to `phreeqcInterface` and the entity name to `phreeqc_interface`. What's your preference? @endJunction @HBShaoUFZ

Reply:
IO and Interface have in this context same meaning to me. I don't see a difference.
The word interface has a more general meaning...

Also the class name should start with capital letter.

Reply:
Selected the first option. PhreeqcIO used where required.